sdk/monitor/monitor-ingestion/test/public/logsIngestionClient.spec.ts

```diff
@@ -51,38 +53,46 @@ describe("LogsIngestionClient live tests", function () {
   });
 
   it("sends empty data", async function () {
-    const result = await client.upload(getDcrId(), "Custom-MyTableRawData", []);
-    assert.equal(result.status, "Success");
+    try {
+      await client.upload(getDcrId(), "Custom-MyTableRawData", []);
+    } catch (e: any) {
+      assert.fail(e);
+    }
```

Comment:
I think mocha should catch any thrown exceptions by default so this is equivalent:

```suggestion

    await client.upload(getDcrId(), "Custom-MyTableRawData", []);

```